### PR TITLE
Add cascading delete of inquiries associated with user when user is d…

### PR DESCRIPTION
**Pull request by RasmusBroborg**
…eleting account. #95.

### CODE REVIEW

File: src/GT.Core/Services/Impl/GTListingInquiryService.cs [hunk 1]
```diff
@@ -89,6 +89,22 @@ public async Task DeleteAsync(string id)
 			}
 		}
 
+		public async Task DeleteInquiriesAssociatedWithUserAsync(string userId)
+		{
+			var associatedInquiryIds = await _listingInquiryRepository.GetAll()
```
Comment (JohanSaisa) [inline src/GT.Core/Services/Impl/GTListingInquiryService.cs:94]: Implement try catch

File: src/GT.Core/Services/Impl/GTListingInquiryService.cs [hunk 2]
```diff
@@ -89,6 +89,22 @@ public async Task DeleteAsync(string id)
...
+				.Where(e => e.ApplicantId == userId)
```
Comment (JohanSaisa) [inline src/GT.Core/Services/Impl/GTListingInquiryService.cs:95]: Implement logger

**Review by JohanSaisa — CHANGES_REQUESTED** [review]
Review comments per dialogue.

File: src/GT.Core/Services/Impl/GTListingInquiryService.cs (hunk 1 above)
Comment (RasmusBroborg) [reply src/GT.Core/Services/Impl/GTListingInquiryService.cs]: Fixed in commit.

File: src/GT.Core/Services/Impl/GTListingInquiryService.cs (hunk 2 above)
Comment (RasmusBroborg) [reply src/GT.Core/Services/Impl/GTListingInquiryService.cs]: Fixed in commit.

**Review by antoncarlgren — APPROVED** [review]

**Review by JohanSaisa — APPROVED** [review]
Approve

**Pull request merged**